docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Add a TensorFlow Lite Modular Service"
+linkTitle: "Add a TensorFlow Lite Modular Service"
+weight: 70
+type: "docs"
+description: "Add a MLModel modular-resource-based service which uses TensorFlow Lite to classify audio samples."
+tags: ["ml", "model training", "services"]
+# SMEs: Andrew Morrow
+---
+
+Viam provides an example [modular resource](extend/modular-resources) written in C++ that extends the [ML model](/services/ml/) service to run any TensorFlow Lite model.
```

Comment:
When reading this intro, it did not become clear to me until line 22 ("This tutorial walks you through...") how this page relates to the mentioned example. Maybe reorder or front-load the "what" of this page a bit more. One idea:
```suggestion
This tutorial guides you through an example [modular resource](extend/modular-resources) written in C++ that extends the [ML model](/services/ml/) service to run any TensorFlow Lite model.
```